imagebuildah/build.go

```diff
@@ -862,6 +930,20 @@ func historyMatches(children []*parser.Node, history []v1.History) bool {
 	return true
 }
 
+// getPolicyContext sets up, initializes and returns a new context for the specified policy
```

Comment:
This feels like a function that should live in a utility class, not here in build.  Besides that, is there a way to get to SignaturePolicyPath from here rather than pulling again?

Reply:
Moved to util. Not sure what you mean by not pulling the SignaturePolicyPath, I don't see where that is happening?

Reply:
I'm not sure about your question about my question!  ;^)   What I was trying to say is we rarely (never?) need to get the signature policy after the first time we get it during the init process. I think that value is set in the SignaturePolicyPath variable,  I'm just not sure if you can get to that variable from this code or not.  If you can, it would see preferable over doing the new function that you put together.  

Then again a ton of reviews today and I maybe off base.